src/morpho-chainlink/MorphoChainlinkOracleV2.sol

```diff
@@ -66,6 +66,9 @@ contract MorphoChainlinkOracleV2 is IMorphoChainlinkOracleV2 {
     /// @param quoteVaultConversionSample The sample amount of quote vault shares used to convert to underlying.
     /// Pass 1 if the quote asset is not a vault. Should be chosen such that converting `quoteVaultConversionSample` to
     /// assets has enough precision.
+    /// @dev Morpho Blue markets have an issue when the price of the oracle can change instantly such that the new price
+    /// is less than the old price multiplied by LLTV*LIF. So in particular, vaults that can receive donations as
+    /// loan/quote assets are problematic.
```

Comment:
```suggestion
    /// @dev Properly configured Morpho Blue markets should ensure that price of the oracle cannot change instantly such that the new price
    /// is less than the old price multiplied by LLTV*LIF. So in particular, vaults that can receive donations shouldn't be used as
    /// loan/quote assets.
```
to avoid the words "issue" & "problematic", when the behavior is part of the spec

Reply:
in practice you can use vaults that can receive donations but you need to price them in an other way. Applied anyway